### PR TITLE
Feature deps check

### PR DESCRIPTION
**Pull request by dan9186**

### CODE REVIEW

**Comment by brimstone** [comment]
I get an interesting error about it disabling the `debian` module. More debugging later.

File: modules/debian/debian [hunk 1]
```diff
@@ -1,3 +1,7 @@
+function _deps(){
+	isosx && myzsh error "This not a linux system" && return 1
```
Comment (brimstone) [inline modules/debian/debian]: Should this be `islinux || myzsh error …` instead?

File: modules/osx/osx [hunk 2]
```diff
@@ -1,2 +1,6 @@
+function _deps(){
+	islinux && myzsh error "This is not an OS X system" && return 1
```
Comment (brimstone) [inline modules/osx/osx]: Same here, `isosx || myzsh error…` ?

**Comment by dan9186** [comment]
Was the error something that was cycling a bunch of times?  Cause I just got that.

**Comment by brimstone** [comment]
Nope, just one line about disabling the debian module. Looks like it's fixed now.

**Comment by dan9186** [comment]
Yay! Then that means more bugs.

**Comment by brimstone** [comment]
i don't have a reason to not :+1: this now.

**Comment by brimstone** [comment]
still :+1: now that `disable` is fixed.

**Pull request merged**